src/components/POLine/Location/validate.js

```diff
@@ -0,0 +1,101 @@
+import React from 'react';
+import { FormattedMessage } from 'react-intl';
+import { get } from 'lodash';
+import {
+  ERESOURCES,
+  INVENTORY_RECORDS_TYPE,
+  OTHER,
+  PHRESOURCES,
+} from '../const';
+
+export const REQUIRED_LOCATION_QUANTITY_MAP_TO_INVENTORY = {
+  [INVENTORY_RECORDS_TYPE.all]: true,
+  [INVENTORY_RECORDS_TYPE.instance]: false,
+  [INVENTORY_RECORDS_TYPE.instanceAndHolding]: true,
+  [INVENTORY_RECORDS_TYPE.none]: false,
+};
+
+export const isLocationPhysicalQuantityRequired = (orderFormat, inventory) => (
+  [...PHRESOURCES, OTHER].includes(orderFormat)
+    ? REQUIRED_LOCATION_QUANTITY_MAP_TO_INVENTORY[inventory]
+    : false
+);
+
+export const isLocationEresourceQuantityRequired = (orderFormat, inventory) => (
+  ERESOURCES.includes(orderFormat)
+    ? REQUIRED_LOCATION_QUANTITY_MAP_TO_INVENTORY[inventory]
+    : false
+);
+
+const LOCATION_MUST_BE_SPECIFIED = 'required';
+
+const getTotalLocationsQuantities = (locations, propName) => {
+  const reducer = (accumulator, d) => accumulator + (d[propName] || 0);
+
+  return locations.reduce(reducer, 0);
+};
+
+export const validateNotNegative = (value) => (value < 0
```

Comment:
this code style is confusing. We do have an eslint rule and you make code harder to read just to bypass it. Please remove braces and format the code as eslint suggests